digiroad2-oracle/src/main/resources/db/migration/V154__Add_Common_History_Seq.sql

```diff
@@ -0,0 +1 @@
+CREATE SEQUENCE COMMON_HISTORY START WITH 1 CACHE 10 INCREMENT BY 1;
```

Comment:
we have duplicate sequence. This one makes the older one deprecated

Reply:
Ossi will remove this one and use the common_history_seq.